ee/dtr/architecture.md

```diff
@@ -11,76 +11,62 @@ Docker Universal Control Plane cluster.
 
 ![](images/architecture-1.svg)
 
-Once you have DTR deployed, you use your Docker CLI client to login, push, and
-pull images.
+Following DTR deployment use the Docker CLI client to login, push, and pull images.
 
-## Under the hood
+## High availability
 
-For high-availability you can deploy multiple DTR replicas, one on each UCP
+To ensure high-availability, deploy multiple DTR replicas, one on each UCP
```

Comment:
I think this should be "each one on its own UCP worker node".  Definitely not each UCP worker node.  This would be a good place to mention that there should be 3, 5 or (maybe) 7 replicas due to raft consensus safety.  We might have details for that documented elsewhere.

@squizzi Do customers run with 5 or 7 nodes?  7 is pushing it, right?